ast/Helpers.h

```diff
@@ -287,8 +288,20 @@ class MK {
         return Hash(loc, std::move(keys), std::move(values));
     }
 
-    static TreePtr Sig(core::LocOffsets loc, TreePtr hash, TreePtr ret) {
-        auto params = Send1(loc, Self(loc), core::Names::params(), std::move(hash));
+private:
+    static TreePtr Params(core::LocOffsets loc, TreePtr recv, Send::ARGS_store args) {
```

Comment:
This helper function makes it easier to construct signatures with user-supplied argument lists.